risk-management-plan.md

```diff
@@ -4,8 +4,13 @@
 
 ## Risk Identification
 
-| Risk | Description | Probability | Potential Loss | Risk Exposure | Risk Category |
-| ---- | ----------- | ----------- | -------------- | ------------- | ------------- |
+| Risk              | Description | Probability | Potential Loss | Risk Exposure | Risk Category |
+| ----------------- | ----------- | ----------- | -------------- | ------------- | ------------- |
+| Volatile Workflow | Given that some of the team members don't know the language, it might affect the workflow as miscommunication is prone to happen | 0.2  | 100 000  | 20 000 | Mission and Goals |
+| Development Team Stability | There are 5 developers that have less than 5 years of experience, they might delay the project as they are learning | 0.3 | 100 000  | 30 000 | Organization Management |
+| Support from the CRM | The CRM might not comply with some requests causing problems with resource allocation or team changes | 0.1  | 50 000  | 5 000 | Organization Management |
+| No Customer Involvement | Not having user tests during development might make the end product not useful, therefore people wont use it | 0.4  | 250 000  | 100 000 | Customer |
+| Scalping of Tickets  | People that scalp tickets to resell will influence the customer experience, they might not get the tickets for a fair price  | 0.4 | 500 000 | 200 000 | Customer |
```

Comment:
Style consistency
```suggestion
| Volatile Workflow | Given that some of the team members don't know the language, it might affect the workflow as miscommunication is prone to happen. | 0.2  | 100,000  | 20,000 | Mission and Goals |
| Development Team Stability | There are 5 developers that have less than 5 years of experience, they might delay the project as they are learning. | 0.3 | 100,000  | 30,000 | Organization Management |
| Support from the CRM | The CRM might not comply with some requests causing problems with resource allocation or team changes. | 0.1  | 50,000  | 5,000 | Organization Management |
| No Customer Involvement | Excluding customers from the testing phase may result in a product that does not align with user expectations or needs, leading to low adoption rates and possible project failure. | 0.4  | 250,000  | 100,000 | Customer |
| Scalping of Tickets  | Scalping could negatively impact customer satisfaction after inflated prices, limited availability, affecting the reputation and revenue.  | 0.4 | 500,000 | 200,000 | Customer |
```